tmt/convert.py

```diff
@@ -411,12 +418,12 @@ def read_nitrate(beaker_task, common_data, disabled):
     # or try to remove if there isn't.
     md_path = os.getcwd() + '/test.md'
     if md_content:
-        write_markdown(md_path , md_content)
+        write_markdown(md_path, md_content)
     else:
         try:
             os.remove(md_path)
             echo(style(f"Test case file '{md_path}' "
-            "successfully removed.", fg='magenta'))
+                       "successfully removed.", fg='magenta'))
```

Comment:
There are just some formatting mistakes, I don't know how did It happen - will fix that!